omero/sysadmins/cli/usergroup.txt

```diff
@@ -21,6 +21,12 @@ enter::
 Additional parameters such as the email address, institution, middle name etc
 can be passed as optional arguments to the :omerocmd:`user add` command.
 
+    $ bin/omero ldap create jsmith 
+
+If you are using ldap as an authentication backend this command will create
+an OMERO user account for `jsmith`. This allows the administrator to add 
```

Comment:
jsmith is marked up in the previous paragraph, if you think it is clearer, could you please add the markup above too.